references.bib

```diff
@@ -451,6 +505,7 @@ @article{1954Coxeter
 title = {Uniform Polyhedra},
 volume = {246},
 year = {1954},
+url={},
```

Comment:
Why is this url empty?

Reply:
But we need a URL for every BibTeX entry.

Reply:
I have added the URL